src/main/java/org/folio/consortia/repository/UserTenantRepository.java

```diff
@@ -14,6 +14,10 @@
 
 @Repository
 public interface UserTenantRepository extends JpaRepository<UserTenantEntity, UUID> {
+
+  @Query("SELECT ut FROM UserTenantEntity ut WHERE ut.tenant.isDeleted IS NULL OR ut.tenant.isDeleted = FALSE")
+  Page<UserTenantEntity> getAll(Pageable pageable);
+
```

Comment:
Seems we need more changes in this repository.
We have SelfController that is called after login and provide available affiliations for user.
![image](https://github.com/folio-org/mod-consortia/assets/25097693/6d96a1dc-237c-43db-b759-91af96afd8a8)
We should not show affiliations for the deleted tenants in Switch affiliation popup.

In UserTenantController we have method
getUserTenants(UUID consortiumId, UUID userId, String username, String tenantId, Integer offset, Integer limit)
that is used 
I think to be consistent we should apply the same filtering for methods getByUsernameAndTenantId() and get() as well as we going to do for method getByUserId() in scope of SelfController changes

Reply:
Applied Filter for GetByUserId(), getByUsernameAndTenantId()